arrow/util.py

```diff
@@ -23,6 +28,44 @@ def is_timestamp(value):
         return False
 
 
+def windows_datetime_from_timestamp(timestamp, tz=None):
+    """Computes datetime from timestamp. Supports negative timestamps on Windows platform."""
+    sec_frac, sec = math.modf(timestamp)
+    dt = datetime.datetime(1970, 1, 1, tzinfo=dateutil.tz.tzutc()) + datetime.timedelta(
+        seconds=sec, microseconds=sec_frac * 1000000
```

Comment:
This should be good to go, but I had a quick question about this line as well: is it ok that we are passing all of the `sec_frac` into microseconds? Or do we have to explicitly break it up into microseconds and milliseconds when passing the parameters into timedelta?

It should be fine, but I wanted to make sure.

Reply:
I think in the end there's no difference between passing `x` milliseconds and 1000 * `x` microseconds to the `timedelta`. The `datetime` will be adjusted by the same value.